packages/compose/src/hooks/use-focus-outside/test/index.js

```diff
@@ -85,56 +95,4 @@ describe( 'useFocusOutside', () => {
 
 		expect( mockOnFocusOutside ).toHaveBeenCalled();
 	} );
-
-	it( 'should not call handler if focus shifts outside the component when the document does not have focus', async () => {
-		// Force document.hasFocus() to return false to simulate the window/document losing focus
-		// See https://developer.mozilla.org/en-US/docs/Web/API/Document/hasFocus.
-		const mockedDocumentHasFocus = jest
-			.spyOn( document, 'hasFocus' )
-			.mockImplementation( () => false );
-		const mockOnFocusOutside = jest.fn();
-		const user = userEvent.setup();
-
-		render(
-			<FocusOutsideComponent onFocusOutside={ mockOnFocusOutside } />
-		);
-
-		// Click and focus button inside the wrapper, then click and focus
-		// a button outside the wrapper.
-		await user.click(
-			screen.getByRole( 'button', { name: 'Button inside the wrapper' } )
-		);
-		await user.click(
-			screen.getByRole( 'button', { name: 'Button outside the wrapper' } )
-		);
-
-		// The handler is not called thanks to the mocked return value of
-		// `document.hasFocus()`
-		expect( mockOnFocusOutside ).not.toHaveBeenCalled();
-
-		// Restore the `document.hasFocus()` function to its original implementation.
-		mockedDocumentHasFocus.mockRestore();
-	} );
-
-	it( 'should cancel check when unmounting while queued', async () => {
```

Comment:
This is the behavior we're changing. We want the callback because focus is moved outside the component.